kodilla-collections/src/main/java/com/kodilla/collections/interfaces/homework/Ford.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.collections.interfaces.homework;
+
+public class Ford implements Car {
+    int speed;
+
+    public Ford(int speed) {
+        this.speed = 0;
```

Comment:
a po co to? W ogole wywalic konstruktor - java nam dostarczy konstruktor domyslny, a pole liczbowe i tak jest na zero ustawiane. Zreszta po co parametr w konstruktorze, skoro z niego nie korzystasz?